packages/circle-demo-webapp/app/components/ui/badge.tsx

```diff
@@ -4,16 +4,15 @@ import * as React from 'react';
 import { cn } from '~/lib/utils';
 
 const badgeVariants = cva(
-  'inline-flex items-center rounded-full border px-2.5 py-0.5 text-xs font-semibold transition-colors focus:outline-none focus:ring-2 focus:ring-ring focus:ring-offset-2',
+  'inline-flex items-center rounded-full border px-2.5 py-0.5 text-xs font-semibold',
   {
     variants: {
       variant: {
-        default:
-          'border-transparent bg-primary text-primary-foreground hover:bg-primary/80',
-        secondary:
-          'border-transparent bg-secondary text-secondary-foreground hover:bg-secondary/80',
-        destructive:
-          'border-transparent bg-destructive text-destructive-foreground hover:bg-destructive/80',
+        default: 'border-transparent bg-primary text-primary-foreground',
+        secondary: 'border-transparent bg-secondary text-secondary-foreground',
+        accent: 'border-transparent bg-accent text-accent-foreground',
+        destructive: 'border-transparent bg-destructive text-destructive-foreground',
+        positive: 'border-transparent bg-positive text-positive-foreground',
```

Comment:
I tried to add green badge but it doesn't look like on the design so it's not in use yet